edx_notifications/server/web/static/edx_notifications/js/views/notification_pane_view.js

```diff
@@ -0,0 +1,54 @@
+var NotificationPaneView = Backbone.View.extend({
+    initialize: function(){
+        /* re-render if the model changes */
+        this.listenTo(this.collection, 'change', this.collectionChanged);
+
+        this.hydrate();
+
+        this.render();
+    },
+
+    hydrate: function() {
+        /* This function will load the bound collection */
+
+        /* add and remove a class when we do the initial loading */
+        /* we might - at some point - add a visual element to the */
+        /* loading, like a spinner */
+        var self = this;
+        self.$el.addClass('loading');
```

Comment:
In Studio we use the class 'ui-loading' so we might want to standardize on that:

https://github.com/edx/edx-platform/blob/master/cms/static/js/views/utils/view_utils.js#L31